src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/GerritNotifier.java

```diff
@@ -68,6 +76,42 @@ public GerritNotifier(IGerritHudsonTriggerConfig config, GerritCmdRunner cmdRunn
         this.parameterExpander = new ParameterExpander(config, hudson);
     }
 
+    /**
+     * Notify changes with the same topic.
+     *
+     * @param event the change based event.
+     * @param command the gerrit command.
+     */
+    private void notifySameTopic(ChangeBasedEvent event, String command) {
+        Topic topic = event.getChange().getTopicObject();
+        if (topic != null) {
+            String pattern = "CHANGE,PATCHSET".replace("CHANGE", event.getChange().getNumber()).replace("PATCHSET",
+                    event.getPatchSet().getNumber());
```

Comment:
This looks very convoluted and slow compared to a simple string concatenation;

```suggestion
            String pattern = new StringBuilder(event.getChange().getNumber()).append(',').append(event.getPatchSet().getNumber()).toString();
```

Reply:
Fixed